### PR TITLE
Add xfixes bindings

### PR DESCRIPTION
**Pull request by Jokler**
I wrote these based on the header on my system so I hope they are fine.

### CODE REVIEW

File: src/xfixes.rs [hunk 1]
```diff
@@ -2,10 +2,104 @@
 // The X11 libraries are available under the MIT license.
 // These bindings are public domain.
 
-use super::xlib::XID;
+use libc::{c_char, c_int, c_short, c_uint, c_ulong, c_ushort};
+use xlib::{Atom, Bool, Cursor, Display, Pixmap, Status, Time, Window, XRectangle, GC, XID};
```
Comment (AltF02) [inline src/xfixes.rs:6]: It doesn't build correctly due to this use statement, a suggestion would be to use `super::`

Cheers

**Review by AltF02 — CHANGES_REQUESTED** [review]
Build fails

File: src/xfixes.rs (hunk 1 above)
Comment (Jokler) [reply src/xfixes.rs]: It's been a while, I don't even remember why I dropped the `super::`.

**Review by AltF02 — APPROVED** [review]
LGTM!

**Pull request merged**